File: Daily Coding/python/union_count.py
#Find the union of two arrays and print the number elements in the resultant array
a = []
print("Enter the length of both arrays: ")
len1 = int(input())
len2 = int(input())
print("Enter the first array: ")
for i in range(len1):
    temp = int(input())
    a.append(temp)
print("Enter the second array: ")
for i in range(len2):
    temp = int(input())
    if temp not in a:
        a.append(temp)
print(f"Length of the union is {len(a)}")

File: Daily Coding/c/union_count.c
// Find the union of two arrays and print the number elements in the resultant array
#include <stdio.h>
#include <stdlib.h>
int main()
{
    int *count = (int*)calloc(sizeof(int),256);
    int a[50], i, len1, len2, temp;
    printf("Enter the length of both the arrays: ");
    scanf("%d %d",&len1,&len2);
    printf("Enter the first array: ");
    for(i=0; i<len1; i++)
    {
        scanf("%d",&a[i]);
        count[a[i]]++;
    }
    printf("Enter the second array: ");
    for(i=0; i<len2; i++)
    {
        scanf("%d",&temp);
        if(count[temp] == 0)
            len1++;
    }
    printf("The length of the union is %d",len1);
    return 0;
}
